### PR TITLE
[ja] updated configure-liveness-readiness-startup-probes.md

### PR DESCRIPTION
**Pull request by Arhell**
en PR https://github.com/kubernetes/website/pull/32221/files
/assign @atoato88
/assign @nasa9084

### CODE REVIEW

File: content/ja/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ Probeの動作としては、kubeletは`cat /tmp/healthy`を対象のコンテ
 このコンテナは、起動すると次のコマンドを実行します:
 
 ```shell
-/bin/sh -c "touch /tmp/healthy; sleep 30; rm -rf /tmp/healthy; sleep 600"
+/bin/sh -c "touch /tmp/healthy; sleep 30; rm -r /tmp/healthy; sleep 600"
```
Comment (atoato88) [inline content/ja/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md:54]: [original PR](https://github.com/kubernetes/website/pull/32221/files#diff-ecfc5eedca9c874f98e389013c37ec5449307be91118305cd18e262df3e077c8R62) changes `-rf` with `-f`.
```suggestion
/bin/sh -c "touch /tmp/healthy; sleep 30; rm -f /tmp/healthy; sleep 600"
```

File: content/ja/examples/pods/probe/exec-liveness.yaml [hunk 2]
```diff
@@ -11,7 +11,7 @@ spec:
     args:
     - /bin/sh
     - -c
-    - touch /tmp/healthy; sleep 30; rm -rf /tmp/healthy; sleep 600
+    - touch /tmp/healthy; sleep 30; rm -r /tmp/healthy; sleep 600
```
Comment (atoato88) [inline content/ja/examples/pods/probe/exec-liveness.yaml:14]: ditto.
```suggestion
    - touch /tmp/healthy; sleep 30; rm -f /tmp/healthy; sleep 600
```

**Review by atoato88 — COMMENTED** [review]
I left some comments, please check those.

File: content/ja/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md (hunk 1 above)
Comment (Arhell) [reply content/ja/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md]: Done

File: content/ja/examples/pods/probe/exec-liveness.yaml (hunk 2 above)
Comment (Arhell) [reply content/ja/examples/pods/probe/exec-liveness.yaml]: Done

**Pull request merged**